File: php/05_functions.php
<?php
function greet($name) {
    return "hello $name";
}

function add($a, $b) {
    return $a + $b;
}

function say($msg = "hi") {
    echo $msg . "\n";
}

echo greet("ulas") . "\n";
echo add(3, 5) . "\n";
say();
say("yo");

File: go/05_functions.go
package main

import "fmt"

func greet(name string) string {
	return "hello " + name
}

func add(a, b int) int {
	return a + b
}

func say(msg string) {
	if msg == "" {
		msg = "hi"
	}
	fmt.Println(msg)
}

func main() {
	fmt.Println(greet("ulas"))
	fmt.Println(add(3, 5))
	say("")
	say("yo")
}
